DGN/Views/Users/Index.cshtml

```diff
@@ -62,4 +73,37 @@
             }
         </tbody>
     </table>
-</div>
\ No newline at end of file
+</div>
+
+@section Scripts {
+    <script src="~/js/UserSearch.js"></script>
+    <script type="text/html" id="hidden-template">
+        <tr>
+            <td>
+                <a href="{email}">
+                    {email}
+                </a>
+            </td>
+            <td>
+                {username}
+            </td>
+            <td>
+                {firstname}
+            </td>
+            <td>
+                {lastname}
+            </td>
+            <td>
+                {birthday}
+            </td>
+            <td>
+                {role}
+            </td>
+            <td>
+                <a href="Users/EditAsAdmin/{id}">Edit</a> |
+                <a href="Users/Profile/{id}">Profile</a> |
+                <a href="Users/Delete/{id}">Delete</a>
```

Comment:
There are some changes from main - not every user will have DELETE in the options, look at this page in main and change it to like it's there.